src/lib/Hydra/Helper/BuildDiff.pm

```diff
@@ -80,8 +85,15 @@ sub buildDiff {
         } else {
             push @{$ret->{new}}, $build if !$found;
         }
-        if (defined $build->buildstatus && $build->buildstatus != 0) {
-            push @{$ret->{failed}}, $build;
+
+        if ($build->finished != 0 && $build->buildstatus != 0) {
+            if ($aborted) {
+                ++$ret->{totalAborted};
```

Comment:
Woah, is that really the right precedence?

Would

```suggestion
                ++($ret->{totalAborted});
```
work for those of us that don't really know Perl?

Reply:
Ah OK, it's a cherry-pick, and more info is in the commit message again.